cava/Cava2HDL.cabal

```diff
@@ -87,7 +87,7 @@ library
                      Byte
                      Decimal
                      Hexadecimal
-                     Numeral
+                     Number
```

Comment:
@blaxill I haven't checked for sure whether it fails on 8.12, but I think this naming change is likely to be an issue.

Reply:
Good point, don't worry about it for now!